app_legacy/Helpers/render/site-award.php

```diff
@@ -132,16 +136,42 @@ function RenderAwardGroup($awards, $title): void
         $counters = RenderCounter($icon, $text, $numItems, $numHidden);
     }
 
-    echo "<div id='" . strtolower(str_replace(' ', '', $title)) . "'>";
+    $visibleAwards = array_filter($awards, fn ($award) => $award['DisplayOrder'] >= 0);
+
+    /**
+     * It is possible for the user to specify they want awards to
+     * always be expanded. This is set via a temporary cookie value.
+     * If it's set, we must never conceal an awards group.
+     * @see public/controlpanel.php
+     */
+    $expandedAwardsCookieName = 'prefers_always_expanded_rewards';
+    $doesPreferAlwaysExpandedRewards =
+        isset($_COOKIE[$expandedAwardsCookieName])
+        && $_COOKIE[$expandedAwardsCookieName] === 'true';
```

Comment:
Instead of using the `$_COOKIE` superglobal (which would result in warnings in some code quality tools) Laravel's `cookie()` helper can be used.

To not encrypt this cookie, which is the default, it has to be excluded explicitly here: https://github.com/RetroAchievements/RAWeb/blob/master/app/Site/Middleware/EncryptCookies.php#L14-L18

Reply:
I am able to set the cookie with `cookie()`, but I am lost on how to read its current value using `cookie()`.

```php
$doesPreferAlwaysExpandedRewards = cookie($expandedAwardsCookieName) === 'true';
```

The above always seems to fail, even if the cookie is set as "true".

Reply:
![Screenshot 2023-03-02 at 6 39 20 PM](https://user-images.githubusercontent.com/3984985/222589711-648578f0-4146-4c4e-8dd2-6e24789e4620.png)

When I do `echo cookie(...)`, the value comes out as being deleted?